src/background/controller/wallet.ts

```diff
@@ -3820,6 +3821,53 @@ export class WalletController extends BaseController {
       await storage.set('currentId', userId);
     }
   };
+
+  getEvmNFTID = async (address: string) => {
```

Comment:
Would you be able to capitalize these in the common way?

```suggestion
  getEvmNftId = async (address: string) => {
```